core/templates/dev/head/pages/exploration_editor/history_tab/CompareVersionsService.js

```diff
@@ -14,7 +14,7 @@
 
 /**
  * @fileoverview Services for the exploration history tab.
- * Service to compare versions of explorations
+ * Service to compare versions of explorations.
  */
```

Comment:
Great! Also I think we don't need the `Services for the exploration history tab.` anymore, right?